corehq/apps/case_search/templates/case_search/case_search.html

```diff
@@ -0,0 +1,138 @@
+{% extends 'style/bootstrap3/base_section.html' %}
+{% load hq_shared_tags %}
+{% load compress %}
+{% load i18n %}
+{% block stylesheets %}
+    <link rel="stylesheet" href="{% new_static 'codemirror/lib/codemirror.css' %}"/>
+    <link rel="stylesheet" href="{% static 'codemirror/addon/fold/foldgutter.css' %}"/>
+{% endblock stylesheets %}
+{% compress js %}
+{% block js %}{{ block.super }}
+    <script src="{% static 'codemirror/lib/codemirror.js' %}"></script>
+    <script src="{% static 'codemirror/mode/javascript/javascript.js' %}"></script>
+    <script src="{% static 'codemirror/addon/fold/foldcode.js' %}"></script>
+    <script src="{% static 'codemirror/addon/fold/foldgutter.js' %}"></script>
+    <script src="{% static 'codemirror/addon/fold/brace-fold.js' %}"></script>
+    <script src="{% static 'case_search/js/case_search.js' %}"></script>
+{% endblock js %}
+{% endcompress %}
+{% block js-inline %}
+    <script>
+     var CaseSearchModel = hqImport('case_search/js/case_search.js');
+     ko.applyBindings(new CaseSearchModel("{% url 'case_details' request.domain '___' %}"), $("#case-search").get(0));
```

Comment:
We've got a [wrapper](https://github.com/dimagi/commcare-hq/blob/422afba20c2afe19b273b48566b7cf0edd15e58e/corehq/apps/style/static/style/js/hq.helpers.js#L130) for this that is the preferred way to apply bindings.  It'd look like:
```javascript
$("#case-search").koApplyBindings(new CaseSearchModel("{% url 'case_details' request.domain '___' %}"))
```
(Basically it just enforces that you're applying it to precisely one element, which you are here, but we've had problems in the past where that's not the case)